runtime/stdlib.js

```diff
@@ -21,17 +21,12 @@
 
 //Provides: raw_array_sub
 function raw_array_sub (a,i,l) {
-  var b = new Array(l);
-  for(var j = 0; j < l; j++) b[j] = a[i+j];
-  return b
+  return a.slice(i, i + l);
```

Comment:
Apparently, `slice` used to be slower (see 1bcded11b03aa2d335ef45073072994b2fc898d4). Is this still the case?

Reply:
According to [this microbenchmark](http://jsben.ch/slRuP), `slice` is three times faster